src/Compilers/Core/Portable/NativePdbWriter/ISymUnmanagedWriter.cs

```diff
@@ -73,9 +73,11 @@ internal static class ISymUnmanagedWriter2Helper
     {
         public static unsafe void DefineConstant2(this ISymUnmanagedWriter2 writer, string name, object value, uint sigToken)
         {
-            VariantStructure variant = new VariantStructure();
-            Marshal.GetNativeVariantForObject(value, new IntPtr(&variant));
-            writer.DefineConstant2(name, variant, sigToken);
+            // Xamarin Bug https://bugzilla.xamarin.com/show_bug.cgi?id=37632
```

Comment:
I think we'll want to just continue to build against PLIB - and disable PDB writing earlier.